pkg/api/v1beta3/types.go

```diff
@@ -252,7 +252,7 @@ type PersistentVolumeSource struct {
 	// NFS represents an NFS mount on the host
 	NFS *NFSVolumeSource `json:"nfs,omitempty" description:"NFS volume resource provisioned by an admin"`
 	// RBD represents a Rados Block Device mount on the host that shares a pod's lifetime
-	RBD *RBDVolumeSource `json:"rbd" description:"rados block volume that will be mounted on the host machine"`
+	RBD *RBDVolumeSource `json:"rbd,omitempty" description:"rados block volume that will be mounted on the host machine"`
```

Comment:
I noticed they were one way while v1beta3 was another.  I thought maybe it was purposeful.  I believe they used to be there.

I'll add 'omitempty' to the other versions, too.